src/utils/generator.js

```diff
@@ -113,6 +126,16 @@ export class GeneratorEntry {
   getDependencies(): void | Array<Generator> {
     invariant(false, "GeneratorEntry is an abstract base class");
   }
+
+  happensAfter(entry: GeneratorEntry): boolean {
```

Comment:
Thinking about it, a correct name would be `notEqualToAndDoesNotHappenBefore`. And similarly for the other function.